Classes/Emogrifier.php

```diff
@@ -1762,46 +1762,22 @@ private function getNodesToExclude(\DOMXPath $xPath)
      */
     public function handleXpathQueryWarnings($type, $message, $file, $line, array $context) // @codingStandardsIgnoreLine
     {
-        // Do not tie this conditional to debug mode as it causes unnecessary output. See further explanation in
-        // ::process() where this exception is caught.
+        $selector = '';
         if (isset($context['cssRule']['selector'])) {
-            throw new \InvalidArgumentException(
-                sprintf(
-                    '%1$s in selector >> %2$s << in %3$s on line %4$u',
-                    $message,
-                    $context['cssRule']['selector'],
-                    $file,
-                    $line
-                ),
-                1509279985
-            );
-        }
-
-        // Catches eventual warnings generated by invalid/unrecognized selectors in method getNodesToExclude()
-        if (isset($context['selectorToExclude'])) {
-            throw new \InvalidArgumentException(
-                sprintf(
-                    '%1$s in selector >> %2$s << in %3$s on line %4$u',
-                    $message,
-                    $context['selectorToExclude'],
-                    $file,
-                    $line
-                ),
-                1511683740
-            );
+            // warnings generated by invalid/unrecognized selectors in getNodesToExclude
```

Comment:
Great simplification!

The only thing to mention is that the comment on this line is not correct. The source of the warning is the `process` method (https://github.com/MyIntervals/emogrifier/blob/e07adc4662359bd977dad6c18dfb314f74365aa9/Classes/Emogrifier.php#L370).

Reply:
Thanks! Fixed.